modules/installation-configuration-parameters.adoc

```diff
@@ -299,9 +305,40 @@ ifdef::osp[]
 |`platform.openstack.lbFloatingIP`
 |An existing floating IP address to associate with the load balancer API.
 |An IP address, for example `128.0.0.1`.
+|====
+
+.Optional {rh-openstack} parameters
+[%header, cols=".^2,.^3,.^5a"]
+|====
+|Parameter|Description|Values
+
+|`compute.platform.openstack
+.additionalNetworkIDs`
+|Additional networks that are associated with compute machines. Allowed address pairs are not created for additional networks.
+|A list of one or more UUIDs as strings, for example `fa806b2f-ac49-4bce-b9db-124bc64209bf`.
```

Comment:
for example,

Reply:
Just repeating what I see elsewhere in the module. I'll fix this here and fix it in other places in a new PR.

Reply:
Created #23257